code/datums/supplypacks/security.dm

```diff
@@ -31,7 +31,7 @@
 	cost = 40
 	containertype = /obj/structure/closet/crate/secure/gear
 	containername = "Armor crate"
-	access_armory //VOREStation Add - Armor is for the armory.
+	access = access_armory //VOREStation Add - Armor is for the armory.
```

Comment:
This line does not belong in this file.

Reply:
Can I clarify why exactly doesnt it belong in that file? Access of the crate ordered from cargo is defined in the supplypack itself. And it overrides the crate's own access restriction.

Reply:
https://github.com/VOREStation/VOREStation/blob/master/.github/CONTRIBUTING.md